Sprint-1/explore/chrome.md

```diff
@@ -9,10 +9,16 @@ Let's try an example.
 
 In the Chrome console,
 invoke the function `alert` with an input string of `"Hello world!"`;
+ *alert("Hello world")
 
 What effect does calling the `alert` function have?
+ *a pop-up dialog box will open
 
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
+ *let myName = prompt("what is your name?")
 
 What effect does calling the `prompt` function have?
+ *a pop-up dialog box will open with input field to enter the input
+
 What is the return value of `prompt`?
+ *prompt return the value in the console but when we store it in variable it shows undifined. we can use console.log(myName) then we can see the return value, 
```

Comment:
- If a user did not enter any input and pressed OK, what value do you expect prompt() will return?
- If a user entered some value but pressed Cancel, what value do you expect prompt() will return?